src/pudl/metadata/fields.py

```diff
@@ -818,14 +870,29 @@
         "description": "Total consumption of fuel in physical unit, year to date. Note: this is the total quantity consumed for both electricity and, in the case of combined heat and power plants, process steam production.",
         "unit": "MMBtu",
     },
+    "fuel_consumed_total_cost": {
+        "type": "number",
+        "description": "Total cost of consumed fuel.",
+        "unit": "USD",
+    },
     "fuel_consumed_units": {
         "type": "number",
         "description": "Consumption of the fuel type in physical unit. Note: this is the total quantity consumed for both electricity and, in the case of combined heat and power plants, process steam production.",
     },
+    "fuel_cost": {
+        "type": "number",
+        "description": "Total fuel cost for plant (in $USD).",
+        "unit": "USD",
+    },
```

Comment:
Might be helpful to explain the difference between `fuel_consumed_total_cost` and `fuel_cost`. Also the fact that only one has the word `total` in it is a little confusing. Also if one of the descriptions says (in USD) then all the cost fields probably should. Unless the metadata has a way to importing the `unit` field as well which would be nice.

Reply:
Moved to #2533